public/app/plugins/panel/heatmap/ExemplarLabelEditor.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { StandardEditorProps } from '@grafana/data';
+import { InlineField, TagsInput } from '@grafana/ui';
+
+export const ExemplarLabelEditor = ({ value, onChange, context }: StandardEditorProps<string[]>) => {
+  const labels = context.options?.exemplars?.labels;
+  const setExemplarLabels = (ls?: string[]) => {
+    onChange(ls);
+  };
+  return (
+    <InlineField
+      label="Labels"
+      tooltip="Add exemplar labels to be displayed on exemplar tooltip. Note that the order of labels added is also important and the same order is displayed on the exemplar tooltip"
+    >
+      <TagsInput placeholder="trace_id" tags={labels} onChange={(ls) => setExemplarLabels(ls)} />
+    </InlineField>
```

Comment:
I noticed a couple of UI inconsistencies here:

1) The UI doesn't scale well, if a user adds too many labels the tags / input go off of the page
![design breaks if too many tags are added](https://github.com/grafana/grafana/assets/22381771/572a66b2-fe3f-4494-b628-e27c18e4eaa6)


2) The input width is tied to the number of tags (ideally it is just constant)
![length of input is tied to number of tags - should just be static](https://github.com/grafana/grafana/assets/22381771/929c8552-c79b-4190-a61d-e67fa0711a85)